---
string FREEDRIVE = freedrive
string SERVOING = servoing
string PLAYBACK = playback
string IDLE = idle
string mode
